.env.sample

```diff
@@ -0,0 +1,7 @@
+# Databse configuration
+DB_HOST="localhost"
+DB_TYPE=mssql
+DB_PORT=1433
+DB_USER_NAME=user1
+DB_PASSWORD='!!!!!!!'
+DB_DATABASE=diffal
```

Comment:
A minus red sign is seen. Maybe it is better to press Enter after the last line of code. @mostafa8026